[cherry-pick] Add documentation for installing MiniKF on AWS (#2368) (#2397)
kubeflow/website: content/en/docs/started/workstation/minikf-aws.md

- Kubeflow
- [Kale](https://github.com/kubeflow-kale/kale), an orchestration and workflow
  tool for Kubeflow that enables you to run complete data science workflows
  starting from a notebook
- Arrikto's [Rok](https://www.arrikto.com/rok-data-management/), a data
  management software for data versioning and reproducibility

### Installing MiniKF on AWS

To install MiniKF on AWS, follow the steps below:

1. Go to the [MiniKF page](https://aws.amazon.com/marketplace/pp/B08MBGH311) on
   the AWS Marketplace.
1. Click on the **Continue to Subscribe** button.

   <img src="/docs/images/minikf-aws/minikf-aws-page.png"
       alt="Continue to Subscribe"
       class="mt-3 mb-3 p-3 border border-info rounded">

1. Click on the **Accept Terms** button.

   <img src="/docs/images/minikf-aws/minikf-aws-accept-terms.png"
       alt="Accept Terms"
       class="mt-3 mb-3 p-3 border border-info rounded">

1. Wait for a few minutes and the click on **Continue to Configuration**.

   <img src="/docs/images/minikf-aws/minikf-aws-continue-configuration.png"
       alt="Continue to Configuration"
       class="mt-3 mb-3 p-3 border border-info rounded">

> 1. Select a **Region**. We recommend you keep the default values for **Delivery

Reviewer: The Google and Msft (?) style guidelines suggest using "you" (second person) - to talk to the users directly. Let's try this, though it uses passive voice instead of the active one. 

```suggestion
1. Select a **Region**. You're recommended to keep the default values for **Delivery
```

   Method** and  **Software Version**. Then click **Continue to Launch**.

   <img src="/docs/images/minikf-aws/minikf-aws-configuration.png"
       alt="Configure MiniKF"
       class="mt-3 mb-3 p-3 border border-info rounded">

> 1. Here you can review your configuration details. Don’t forget to view the

Reviewer: Instead of "here you can.." maybe it'd be better to state what the reader is using to review the config.

For instance: `In the **Configuration details** window, you can review your ...`

   **Usage Instructions** and keep them handy, as you are going to need them
   later.

   <img src="/docs/images/minikf-aws/minikf-aws-configuration-details.png"
       alt="Configuration Details"
       class="mt-3 mb-3 p-3 border border-info rounded">

1. You can also select how to launch MiniKF or change the EC2 instance type. We

>    recommend you leave the default value for **Choose Action**. We recommend

Reviewer: ```suggestion
   recommend you leave the default value for **Choose Action**. You're recommended
```

(as before)

   that you keep the default **EC2 Instance Type** (m5.2xlarge) or choose an

>    even more powerful instance. Choosing a VM with reduced specs may make it

Reviewer: ```suggestion
   even more powerful instance. Choosing a VM with reduced computing resources may make it
```

WDYT?

Reviewer: (Or "compute", which is less formal but widely used in the community)

>    impossible to train ML models.

Reviewer: ```suggestion
   very challenging to train machine learning (ML) models.
```

   <img src="/docs/images/minikf-aws/minikf-aws-instance-type.png"
       alt="Select Intance Type"
       class="mt-3 mb-3 p-3 border border-info rounded">

1. Choose an existing **VPC** or create a new one. If you create a new one, make
   sure you click on the **refresh** icon to update the contents of the list, so
   that your new VPC appears in it.

   <img src="/docs/images/minikf-aws/minikf-aws-security-settings-vpc.png"
       alt="Select a VP or create a new one"
       class="mt-3 mb-3 p-3 border border-info rounded">

1. Choose an existing **Subnet** or create a new one. If you create a new one,
   make sure you click on the **refresh** icon to update the contents of the
   list, so that your new subnet appears in it.

   <img src="/docs/images/minikf-aws/minikf-aws-security-settings-subnet.png"
       alt="Select a Subnet or create a new one"
       class="mt-3 mb-3 p-3 border border-info rounded">

> 1. We strongly recommend to create a new security group. Otherwise you may not

Reviewer: ```suggestion
1. You're strongly recommended to **create a new security group**. Otherwise you may not
```

(Let's emphasize this for users who're just scanning the steps.)
